app/services/step_content_parser.rb

```diff
@@ -58,7 +58,9 @@ def safely_parse_path(path)
   end
 
   def relative_paths(content)
-    all_links_in_content(content).select { |href| href.match(/^\/[a-z0-9]+.*/i) }
+    all_links_in_content(content)
+      .map { |url| strip_govuk_prefix(url) }
```

Comment:
We have a lot of regular expressions defined as constants (e.g. `LINK_REGEX`).  It might be a good idea to define `GOVUK_PREFIX_REGEX` as a constant, then use it here?

Reply:
Thanks for the suggestion. Now done 👍 

Was going to do the same for the line below it, but got into a muddle (called it `ABSOLUTE_URL_REGEX`, which is effectively what we're trying to do here, but the regex isn't _really_ an absolute URL regex, so was a bit of a rabbit hole. Would end up with something like `NOT_A_SLUG_REGEX`, which just doesn't read easily!)